server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/node/Tree.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.index.compositeindex.datacube.startree.node;
+
+/**
+ * Interface for star-tree.
+ *
+ * @opensearch.experimental
+ */
+public interface Tree {
+
+    /**
+     * Fetches the root node of the star-tree.
+     * @return the root of the star-tree
+     */
+    StarTreeNode getRoot();
```

Comment:
Lets remove this class. Lets resolve comments post addressal so that we don't miss any comments.

Reply:
removed interface